node/src/command.rs

```diff
@@ -48,7 +50,10 @@ fn load_spec(
 		"stagenet" => Ok(Box::new(chain_spec::ChainSpec::from_json_bytes(
 			&include_bytes!("../../specs/stagenet/parachain-embedded-specs-v6.json")[..],
 		)?)),
-		"dev" | "development" => Ok(Box::new(chain_spec::development_chain_spec())),
+		"dev" | "development" => Ok(Box::new(chain_spec::development_chain_spec(
+			mnemonic.clone(),
+			accounts,
```

Comment:
Here we should use the known everywhere development spec.

Alternatively, `development_chain_spec` could take an `Option<_>` and if you provide none, hten it defaults to the development chain spec.

Reply:
the development_chain_spec indeed takes an option and if none is provided it defaults to the default mnemonic

Reply:
Oh yeah, it does. That's great. So here you should put `None`.